src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -221,20 +221,19 @@ public LocalTrack GetLocalTrack(string filename, Artist artist)
         public LocalTrack GetLocalTrack(string filename, Artist artist, ParsedTrackInfo folderInfo)
         {
             ParsedTrackInfo parsedTrackInfo;
+            Album album = null;
 
 
             if (folderInfo != null)
             {
                 parsedTrackInfo = folderInfo.JsonClone();
                 parsedTrackInfo.Quality = QualityParser.ParseQuality(Path.GetFileName(filename), null, 0);
-            }
-
-            else
+            } else
             {
                 parsedTrackInfo = Parser.ParseMusicPath(filename);
             }
 
-            if (parsedTrackInfo == null || parsedTrackInfo.AlbumTitle.IsNullOrWhiteSpace())
+            if ((parsedTrackInfo == null || parsedTrackInfo.AlbumTitle.IsNullOrWhiteSpace()) && parsedTrackInfo.ReleaseMBId.IsNullOrWhiteSpace())
```

Comment:
Parenthesis are off, should be `if (parsedTrackInfo == null || (parsedTrackInfo.AlbumTitle.IsNullOrWhiteSpace() && parsedTrackInfo.ReleaseMBId.IsNullOrWhiteSpace()))`

We should always kick out if `parsedTrackInfo == null`, or we should also kick out if `AlbumTitle` and `ReleaseMBId`